DEPS

```diff
@@ -1011,7 +1007,6 @@ hooks = [
       'python3',
       'engine/src/build/mac/find_sdk.py',
       '--as-gclient-hook',
-      Var('mac_sdk_min')
```

Comment:
curious what does `Var` mean

Reply:
It's how you refer to a variable defined elsewhere in the DEPS file.